websites/T/Trakt/presence.ts

```diff
@@ -0,0 +1,169 @@
+			"/calendars/my/premieres": {
+				details: "Browsing my calendar",
+				state: "Premieres",
+			},
+			"/calendars/my/new-shows": {
+				details: "Browsing my calendar",
+				state: "New shows",
+			},
+			"/calendars/my/movies": {
+				details: "Browsing my calendar",
+				state: "Movies",
+			},
+			"/calendars/my/dvd": {
+				details: "Browsing my calendar",
+				state: "DVD & Blu-ray",
+			},
+			"/calendars/shows": {
+				details: "Browsing calendar",
+				state: "Shows",
+			},
+			"/calendars/premieres": {
+				details: "Browsing calendar",
+				state: "Premieres",
+			},
+			"/calendars/new-shows": {
+				details: "Browsing calendar",
+				state: "New shows",
+			},
+			"/calendars/movies": {
+				details: "Browsing calendar",
+				state: "Movies",
+			},
+			"/calendars/dvd": {
+				details: "Browsing calendar",
+				state: "DVD & Blu-ray",
+			},
+			"/discover": { details: "Finding new shows/movies" },
+			"/apps": { details: "Browsing apps" },
+			"/vip": { details: "Browsing VIP features" },
+			"/settings": { details: "Browsing settings" },
+			"/auth/signin": { details: "Registering on Trakt" },
+			"/auth/login": { details: "Logging in to Trakt" },
+			"/widgets": { details: "Browsing widgets" },
+		};
+
+	for (const [pathname, data] of Object.entries(pages))
+		if (path.includes(pathname)) presenceData = { ...presenceData, ...data };
+
+	if (path.includes("/users")) {
+		const username = document.querySelector(
+			"#avatar-wrapper >  .emojis-supported > a"
+		).textContent;
+		if (path.includes("/history"))
+			presenceData.details = `Browsing ${username}'s history`;
+		else if (path.includes("/progress"))
+			presenceData.details = `Browsing ${username}'s progress`;
+		else if (path.includes("/collection"))
+			presenceData.details = `Browsing ${username}'s collection`;
+		else if (path.includes("/ratings"))
+			presenceData.details = `Browsing ${username}'s ratings`;
+		else if (path.includes("/lists"))
+			presenceData.details = `Browsing ${username}'s lists`;
+		else if (path.includes("/comments"))
+			presenceData.details = `Browsing ${username}'s comments`;
+		else if (path.includes("/network" || "/network/friends"))
+			presenceData.details = `Browsing ${username}'s friends`;
+		else presenceData.details = `Browsing ${username}'s profile`;
+	} else if (path.includes("/shows/")) {
+		if (path.includes("/episodes/")) {
+			presenceData.details = "Browsing episode";
+			presenceData.state = `${
+				document
+					.querySelector(
+						"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h2 > a:nth-child(1)"
+					)
+					.textContent.split(":")[0]
+			} -
+		${
+			document.querySelector(
+				"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1 > span.main-title-sxe"
+			).textContent
+		}`;
+			presenceData.buttons = [{ label: "Check episode", url: document.URL }];
```

Comment:
Please use this on all other instances
```suggestion
			presenceData.buttons = [{ label: "Check episode", url: document.location.href }];
```